lonestar/mining/cpu/subgraph-listing/CMakeLists.txt

```diff
@@ -2,3 +2,5 @@ add_executable(subgraph-listing-cpu sgl.cpp)
 add_dependencies(apps subgraph-listing-cpu)
 target_link_libraries(subgraph-listing-cpu PRIVATE Galois::pangolin LLVMSupport)
 install(TARGETS subgraph-listing-cpu DESTINATION "${CMAKE_INSTALL_BINDIR}" COMPONENT apps EXCLUDE_FROM_ALL)
+
```

Comment:
nit: just some whitespace things...?

Reply:
Yah, that slipped in along the way either from a weird merge or my initial attempts at adding a test for that app. I'll remove it.